ckanext/georchestra/controller.py

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+import six
+
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+import ckan.plugins.toolkit as t
+
+render = base.render
+
+
+class GeorchestraController(base.BaseController):
+
+    def georchestra_logout(self):
+        '''Override logout with geOrchestra's logout'''
+        return h.redirect_to(u'/logout')
+
+    def georchestra_login(self):
+        '''Override login with geOrchestra's login pattern'''
+        referrer = six.text_type(t.request.referrer, encoding='utf-8')
```

Comment:
I thought it has only one «r» but looks like both are available in `request`

![Screenshot from 2019-04-18 13-12-49](https://user-images.githubusercontent.com/21686/56357368-dde4c400-61db-11e9-9630-9aee0dc76274.png)